src/gwc/src/main/java/org/geoserver/gwc/layer/DefaultTileLayerCatalog.java

```diff
@@ -109,22 +109,22 @@ public boolean accept(Resource res) {
         });
 
         LOGGER.info("Loading tile layers from " + baseDir.path());
-        for (Resource res : tileLayerFiles) {
+        tileLayerFiles.parallelStream().forEach(res -> {
             GeoServerTileLayerInfoImpl info;
             try {
                 info = depersist(res);
             } catch (Exception e) {
                 LOGGER.log(Level.SEVERE, "Error depersisting tile layer information from file "
                         + res.name(), e);
-                continue;
```

Comment:
Gah sorry, this really needs to be removed (long builds, I was playing with parallelizing the DefaultTileLayerCatalog loading with streams, and this change is a leftover. Will remove.